### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by Jaybsoni**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,27 +1,18 @@
...
-### Documentation
-
-### Bug fixes
-
 ### Contributors
 
 This release contains contributions from (in alphabetical order):
 
-Guillermo Alonso-Linaje, Romain Moyard, Tanner Rogalsky, Antal Száva
+Guillermo Alonso-Linaje, Romain Moyard, Jay Soni, Antal Száva, Tanner Rogalsky
```
Comment (antalszava) [inline CHANGELOG.md:15]:
```suggestion
Guillermo Alonso-Linaje, Romain Moyard, Tanner Rogalsky, Jay Soni, Antal Száva
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,27 +1,18 @@
...
 
 * Add two devices for runtime programs and one VQE runtime program solver.
   [(#157)](https://github.com/PennyLaneAI/pennylane-qiskit/pull/157)
-  
-### Breaking changes
-
-### Improvements
 
 * Improved the login flow when IBMQ tokens are specified as environement variables.
   [(#169)](https://github.com/PennyLaneAI/pennylane-qiskit/pull/169)
```
Comment (antalszava) [inline CHANGELOG.md:10]:
```suggestion

### Documentation

* Improved quality of docstrings in the library.
  [(#174)](https://github.com/PennyLaneAI/pennylane-qiskit/pull/174)

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,27 +1,23 @@
...
 
 ### Documentation
 
-### Bug fixes
+* Improved quality of docstrings in the library.
```
Comment (antalszava) [inline CHANGELOG.md:13]:
```suggestion
* Improved the quality of docstrings in the library.
```

**Review by antalszava — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,27 +1,23 @@
...
+# Release 0.21.0
 
 ### New features since last release
 
 * Add two devices for runtime programs and one VQE runtime program solver.
   [(#157)](https://github.com/PennyLaneAI/pennylane-qiskit/pull/157)
-  
-### Breaking changes
-
-### Improvements
```
Comment (Jaybsoni) [inline CHANGELOG.md:7]:
```suggestion

## Improvements

```

Comment (Jaybsoni) [reply CHANGELOG.md:7]: I guess I accidentally removed the Improvements heading here

File: CHANGELOG.md (hunk 1 above)
Comment (Jaybsoni) [reply CHANGELOG.md]: Lol how did I mess that up!

File: requirements.txt [hunk 5]
```diff
@@ -1,6 +1,6 @@
 qiskit>=0.32
 mthree>=0.17.2
-git+https://github.com/PennyLaneAI/pennylane.git
+pennylane>=0.20
```
Comment (Jaybsoni) [inline requirements.txt:3]: @antalszava, do you think the failure is because we actually require the latest version of PL for this ?

Comment (antalszava) [reply requirements.txt:3]: Yes, see here https://github.com/PennyLaneAI/pennylane-qiskit/pull/177/files.

Changed the test to consider this.

Comment (Jaybsoni) [reply requirements.txt:3]: Ahh I see !

File: tests/test_sample.py [hunk 6]
```diff
@@ -201,6 +202,7 @@ def test_pauliz_hadamard_pauliy(self, theta, phi, varphi, device, shots, tol):
         ) / 4
         assert np.allclose(var, expected, **tol)
 
+    @flaky(max_runs=5, min_passes=3)
```
Comment (Jaybsoni) [inline tests/test_sample.py:205]: Did the consistency of this test drop somehow with the new release of PL and qiskit? that's pretty weird no?

File: requirements.txt (hunk 5 above)
Comment (Jaybsoni) [reply requirements.txt:3]: Nice catch!

File: tests/test_sample.py (hunk 6 above)
Comment (antalszava) [reply tests/test_sample.py:205]: I don't think so. Note that [we were pulling PennyLane v0.20.0](https://github.com/PennyLaneAI/pennylane-qiskit/runs/5100591876?check_suite_focus=true#step:5:28) when the CI check was failing and the sampling logic was not affected with the latest changes of PennyLane-Qiskit.

This test has been flaky in the past too, although usually, 1 re-run sufficed.

Comment (Jaybsoni) [reply tests/test_sample.py:205]: Ahh okay,

**Pull request merged**